lib/vp9.py

```diff
@@ -29,10 +29,12 @@ def __init__(self, name='vp9'):
     super(Vp9Codec, self).__init__(name)
     self.extension = 'webm'
     self.option_set = encoder.OptionSet(
-      encoder.IntegerOption('cpu-used', 0, 16),
+      encoder.IntegerOption('cpu-used', 0, 8),
```

Comment:
What does this cpu-used change mean? The description in http://www.webmproject.org/docs/encoder-parameters/ seems a bit cryptic to me. Which is fancy words for that I don't understand it. :-)